### ISSUE TITLE
question about bpe and inference OOM

### ISSUE THREAD

**Issue by ZeroneBo** [carboncoo/mask-align#14]
I'm confused when to use BPE: 
In [preprocess](https://github.com/THUNLP-MT/THUMT/blob/master/docs/walkthrough.md#data-preparation) , there said the valid and test‘s tgt needn't bpe and src need bpe.
But in [example.config](https://github.com/THUNLP-MT/Mask-Align/blob/a04aba513b3a31ea48070eb8d5b555f2097cd1b8/thualign/configs/user/example.config#L8C1-L8C1) , the test both src and tgt applied bpe.
I want to know when and which file in train, valid, test, src or tgt should use bpe, and which needn't to apply bpe.

Annother question, when I inference with the subset of training data, batch_size is setted to 1, the GPU is always OOM. But when train model use full training data, batch_size is 4000, the GPU isn't OOM. I am curious about how to solve it or use only CPU to inference?

Thanks for any helpful suggestions.
